resources/js/Events/Components/Core/EventTree.jsx

```diff
@@ -0,0 +1,281 @@
+class EventTree {
+  constructor(props) {
+    Object.assign(this, props);
+  }
+
+  static make(events, source) {
```

Comment:
Add doc comment for parameters here.
```suggestion
  /**
   * @param {Object} events Event Data object
   * @param {string} source Top level event group (HEK, CCMC, RHESSI, etc)
   */
  static make(events, source) {
```